### DISCUSSION TITLE
Expansion header

### DISCUSSION THREAD

**Discussion by badass-boss** [ZuluSCSI/ZuluSCSI-Pico-OSHW#3] [General]
Does this hardware revision offer expansion header? Is this possible to attach DAC Attack board to it?

**Comment by aperezbios** [comment]
@badass-boss it does not. In order for there to be sufficient I/O's for the DAC Attack add-on board, you must be using a ZuluSCSI RP2040 (non-Pico) board, as the pinouts are slightly different. The RP2040 is a great chip, but it comes with limitations, particularly on the limited I/O front.

**Comment by therealjbenam** [comment]
@aperezbios 
Hi, I was wondering if an RP2350 with its additional I/Os might finally allow audio to happen on the ZuluSCSI Pico?
